doc/man3/CMS_EnvelopedData_create.pod

```diff
@@ -0,0 +1,48 @@
+=pod
+
+=head1 NAME
+
+CMS_EnvelopedData_create - Create CMS envelope
+
+=head1 SYNOPSIS
+
+ #include <openssl/cms.h>
+
+ CMS_ContentInfo *CMS_EnvelopedData_create(const EVP_CIPHER *cipher);
+
+=head1 DESCRIPTION
+
+The B<CMS_ContentInfo>  
+
+CMS_EnvelopedData_create() creates a B<CMS_ContentInfo> structure with
+a type B<NID_pkcs7_enveloped>.
+
+The recipients can be added afterwards and the CMS_final() needs to be called
```

Comment:
A reference to the function to add recipients would be a useful addition.

Reply:
Added a reference in the same way as it is in `CMS_encrypt()` docs.